solanaj/src/main/java/org/p2p/solanaj/utils/TweetNaclFast.java

```diff
@@ -820,18 +820,18 @@ public static KeyPair keyPair() {
             crypto_sign_keypair(kp.getPublicKey(), kp.getSecretKey(), false);
             return kp;
         }
-        public static KeyPair keyPair_fromSecretKey(byte [] secretKey) {
+        public static KeyPair keyPair_fromSecretKey(byte [] keypair) {
```

Comment:
fromSecretKey but you pass keypair, not ok

Reply:
too much to refactor in this PR. 
This should be in tech debt